.gitignore

```diff
@@ -15,6 +15,7 @@ TAGS
 tags
 *.DS_Store
 *.vagrant
+cscope.out
```

Comment:
Seems like something to do in another (trivial) PR. Did you meant to include this change?

Reply:
These are just the equivalent of ctags files, I didn't figure it was worth opening a separate PR for that which is why I rolled it in here. I can move it if you think it's appropriate.

Reply:
I'm OK with it but in the future it's better to make a separate PR (in this case it would be a trivial PR and wouldn't need review).